supertokens_python/utils.py

```diff
@@ -175,3 +179,43 @@ def execute_async(mode: str, func: Callable[[], Coroutine[Any, Any, None]]):
 
 def frontend_has_interceptor(request: BaseRequest) -> bool:
     return get_rid_from_request(request) is not None
+
+
+def deprecated_warn(msg: str):
+    warnings.warn(msg, DeprecationWarning, stacklevel=2)
+
+
+def handle_httpx_client_exceptions(e: Exception, input_: Dict[str, Any]):
+    if isinstance(e, HTTPStatusError):
+        res: Response = e.response  # type: ignore
+        log_debug_message("Error status: %s", res.status_code)  # type: ignore
+        log_debug_message("Error response: %s", res.json())
+    else:
+        log_debug_message("Error: %s", e)
+
+    log_debug_message("Logging the input:")
+    log_debug_message("%s", json.dumps(input_))
+
+
+def humanize_time(ms: int) -> str:
```

Comment:
Have you written tests for this? Where are they?

Reply:
Removed it. It's more sensible to keep it in https://github.com/supertokens/supertokens-python/pull/162/